src/Workspaces/CSharp/Portable/CodeGeneration/EventGenerator.cs

```diff
@@ -199,6 +200,14 @@ private static BlockSyntax GenerateBlock(IMethodSymbol accessor)
                         tokens.Add(SyntaxFactory.Token(SyntaxKind.StaticKeyword));
                     }
 
+                    // An event is readonly if its accessors are readonly.
+                    // If one accessor is readonly and the other one is not,
+                    // the event is malformed and cannot be properly displayed.
```

Comment:
Small request: please link to the tracking bug in the comment regarding the add method being checked specifically.

Reply:
Done